README.md

```diff
@@ -1,30 +1,35 @@
-Readme
-======
-
 OCaml bindings for ZMQ 4.0
+==========================
 
 Requirements
 ------------
-* [OCaml >= 3.12](http://caml.inria.fr/)
+
+* [OCaml >= 3.12](https://ocaml.org/)
 * [Findlib](http://projects.camlcity.org/projects/findlib.html)
 * [ocaml-uint](https://github.com/andrenth/ocaml-uint)
 * [ZMQ 4.0.*](http://www.zeromq.org/intro:get-the-software)
-* [oasis](http://oasis.forge.ocamlcore.org/)
```

Comment:
These dependencies are not needed. Just refer to opam,
If you want them there, please refer to the libries listed in zmq.opam.